docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/hetero-execution.rst

```diff
@@ -90,6 +100,30 @@ It does not take into account device peculiarities such as the inability to infe
          :language: cpp
          :fragment: [compile_model]
 
+Pipeline parallelism (preview)
```

Comment:
@kblaszczak-intel  Is it correct to indicate this is a preview feature?

Reply:
It surely is